changelog.md

```diff
@@ -1,3 +1,17 @@
+# 4.1.2 / 2022-02-14
```

Comment:
make sure you update the date here @raphj

Reply:
```suggestion
# 4.1.2 / 2022-02-15
```